.copyrightignore

```diff
@@ -0,0 +1,34 @@
+# *******************************************************************************
+# Copyright (c) 2018, 2018 IBM Corp. and others
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+# *******************************************************************************/
+#
+# *******************************************************************************
+#  THE FOLLOWING ARE EXAMPLES OF SYNTAXES copyrightCheck SUPPORTS:
+#  1.**/jobs/**  # path contains "jobs" folder, but not root directory
+#  2./doc  #root root level directory and file named "doc"
+#  3./debugtools/**/*.ignore  # all .ignore file in the root directory debugtools
+#  4./*.ignore #all root directory .ignore files
+#  5.**/jobs/pipelines/  #pipeline folder follows closely with jobs not in root directory
+#  6.*.ignore  #all ignore files only
+#  7.**/jobs/**/pipelines/  #directory path contains both jobs and pipelines not root directory
+#  8.*job*  # all things that contains characters in between stars
+# *******************************************************************************/
+
+.copyrightignore
```

Comment:
OK, will remove the `.copyrightignore` file, but this then leaves the file empty other than the comments. Do you have any files / paths that should be added to ignore.

Reply:
> Do you have any files / paths that should be added to ignore.

I believe so, any file that doesn't already have a copyright. I could write/run a little code later to produce a list.

Reply:
I'm fine with an initially blank file. Pete or others can add to it in the future but at least the template is there with instructions.

Reply:
I found 389 OpenJ9 files without a copyright, but since these files tend not to be modified, I don't think there is any rush to add ignores for any of them. I'm happy to deliver a blank file in this PR and modify it later.